### ISSUE TITLE
No keyboard focus state on search submit button

### ISSUE THREAD

**Issue by joedolson** [WordPress/wporg-mu-plugins#148]
Pretty self-explanatory, I think. The submit button for the search form does not have any keyboard focus state.

![image](https://user-images.githubusercontent.com/1686451/152589991-fcdcf3a1-0db0-4abd-9cf2-a6f22eb802c7.png)

**Comment by ryelle** [comment]
The search button isn't actually _visible_ at all. Switching this so that the icon is the search button would help here, and could also help with #99, some work was started in a branch mentioned in [this comment](https://github.com/WordPress/wporg-mu-plugins/issues/99#issuecomment-1015774170).
